tools/report-converter/codechecker_report_converter/cppcheck/analyzer_result.py

```diff
@@ -76,7 +76,7 @@ def _add_report_hash(self, plist_data):
                                          files[diag['location']['file']])
                 diag['issue_hash_content_of_line_in_context'] = report_hash
 
-    def _write(self, file_to_plist_data, output_dir):
+    def _write(self, file_to_plist_data, output_dir, file_name):
```

Comment:
This function overrides the function from the base class: https://github.com/csordasmarton/codechecker/blob/analyze_makefile_output/tools/report-converter/codechecker_report_converter/analyzer_result.py#L138 and it has this parameter. It will throw an exception if we remove this parameter from here.

Reply:
ok, thanks